src/main/java/apaintus/models/shapes/TextBox.java

```diff
@@ -28,7 +27,8 @@ public TextBox(ShapeAttributes shapeAttributes) {
     public void update(ShapeAttributes shapeAttributes) {
         super.update(shapeAttributes);
 
-        //text = shapeAttributes.getText();
+        if (shapeAttributes.getText() != null )
```

Comment:
Est-ce que le textArea peut renvoyer null si rien n'est écrit?

Reply:
non il envoit un string vide

Reply:
Le if not null pourrait être retiré si tu mets un text avec withText dans la factory. À ta discrétion.